worker/src/Channel/Request.cpp

```diff
@@ -112,6 +125,8 @@ namespace Channel
 	{
 		MS_TRACE();
 
+		MS_ASSERT(Request::channel != nullptr, "channel unset");
```

Comment:
Let's replace it with `MS_ASSERT(Request::channel, "channel unset");` and let's remove the empty line between both `MS_ASSERT()`s.